robosherlock/src/segmentation/src/PointCloudClusterExtractor.cpp

```diff
@@ -410,7 +404,17 @@ class PointCloudClusterExtractor : public DrawingAnnotator
     ecc->setInputCloud(cloud);
     ecc->setLabels(input_labels);
     //TODO commented that out i think we have to fix the error
-    ecc->setExcludeLabels(exclude_labels);
+    #if PCL_VERSION_COMPARE(<,1,9,0)
+        std::vector<bool> ignore_labels;
+        ignore_labels.resize(2);
+        ignore_labels[0] = true;
+        ignore_labels[1] = false;
+        ecc->setExcludeLabels(ignore_labels);
+    #else
+        boost::shared_ptr<std::set<uint32_t>> exclude_labels=boost::make_shared<std::set<uint32_t>>();
+        exclude_labels->insert(1);
```

Comment:
Is there a specific reason why this is called exclude_labels instead of ignore_labels like in the other block?
If not, could you please unify the variable names?

Reply:
No, there is no specific reason for doing it. So I just make it to ignore_labels.